diff --git a/dist/dist.dev.js b/dist/dist.dev.js
index 3b509c09deaa7daf456840af0a867aed4256f438..dcbee26b0949f4050ef47cd2c11b9cd1dc3168eb 100644
--- a/dist/dist.dev.js
+++ b/dist/dist.dev.js
@@ -34728,7 +34728,7 @@ var<uniform> layer: LayerUniforms;
 `
   );
   var uniformBlock = `layout(std140) uniform layerUniforms {
-  uniform float opacity;
+  float opacity;
 } layer;
 `;
   var layerUniforms = {
diff --git a/dist/index.cjs b/dist/index.cjs
index 966d28dd008a077f22be496d5cb3b89a925531bf..426c9197b84f275c1daa9e16e65c1d6e71938944 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -270,7 +270,7 @@ var<uniform> layer: LayerUniforms;
 `
 );
 var uniformBlock = `layout(std140) uniform layerUniforms {
-  uniform float opacity;
+  float opacity;
 } layer;
 `;
 var layerUniforms = {
diff --git a/dist/shaderlib/misc/layer-uniforms.js b/dist/shaderlib/misc/layer-uniforms.js
index 9ad93fba65edbb1a7017d4ce46ea393841b3b0e7..9cb176ea83277e54396d84461273d79d77c1ef30 100644
--- a/dist/shaderlib/misc/layer-uniforms.js
+++ b/dist/shaderlib/misc/layer-uniforms.js
@@ -11,7 +11,7 @@ var<uniform> layer: LayerUniforms;
 `;
 const uniformBlock = `\
 layout(std140) uniform layerUniforms {
-  uniform float opacity;
+  float opacity;
 } layer;
 `;
 export const layerUniforms = {
